_posts/2025-10-03-meeting-updates.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: October Updates
+subtitle: Parks and Recreation Commission Meeting Notes
+cover-img: /assets/img/cutter-reinhart-poster-up.jpg
+thumbnail-img: /assets/img/cutter-reinhart-poster.jpg
+share-img: /assets/img/cutter-reinhart-poster.jpg
+author: John Ibsen
+---
+
+It's been amazing to see how much the community cares about this little corner of Arlington! The petition now has 190+ signatures of residents of Arlington and even from neighboring towns.
+
+You may have also seen our sign up to join the Friends of Cutter / Reinhart Park.
+
+![Photo of poster]({{ '/assets/img/cutter-reinhart-poster.png' | relative_url }}){: .mx-auto.d-block :}
+
+## The Sept 30th Parks and Rec Commission Meeting
+
+At the Septemter 30th meeting, several of the community members were present to listen and speak about the importance of the Cutter / Reinhart playground. Leading up to this meeting, community member Alex was able to get in touch with the original manufacture of the playground. The Town and the manufacturer have shared notes and communicated. Even after viewing detailed photos and the prior inspection report, the manufacturer believes that the damage *is* repairable. The Town however must confer with Town Counsel to determine legal liability given that our CPSI (Certified Playground Safety Inspector) has declared it un-repairable.

```

Comment:
"given that the CPSI (certified Playground Safety Inspector) that the town of Arlington hired in 2019 to do an inspection believes its unrepairable based on photos sent in 2025 after residents filed complaints about the play structure."